src/components/DashboardLaunchButton/index.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { logError } from '@edx/frontend-platform/logging';
+import { Button } from 'react-paragon-topaz';
```

Comment:
Could you please add a space here:

```suggestion
import { Button } from 'react-paragon-topaz';

```